README.md

```diff
@@ -71,6 +72,22 @@ use {
 }
 ```
 
+## [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+{
+    "jay-babu/mason-null-ls.nvim",
+    event = { "BufReadPre", "BufNewFile" },
+    dependencies = {
+      "williamboman/mason.nvim",
+      "jose-elias-alvarez/null-ls.nvim",
```

Comment:
Would this be more complete?
```suggestion
      {"williamboman/mason.nvim", config = true },
      {"jose-elias-alvarez/null-ls.nvim", config = true },
```

Reply:
The use of `config = true` is for a situation where we want only the desired plugins to be activated and the person does not have a special configuration for `mason` and `null-ls`.

Reply:
@jay-babu I agree with @Pakrohk. We should skip this because with `mason.nvim` and `null-ls` in most cases user will need custom config which should be handled in:

```lua
config = function()
      require("your.null-ls.config") -- require your null-ls config here (example below)
    end,
```

which I have already added to README.md

Reply:
in lazy.nvim, you **do not** need to declare your dependencies. approved regardless